project3.0/index.html

```diff
@@ -66,40 +96,63 @@ <h2>Creating pixel perfect icons in Figma</h2>
+                    <div class="works-img">
+                        <img src="img/poster2.jpg" alt="img2">
+                    </div>
+                    <div class="works">
+                        <h2>Vibrant Portraits of 2020</h2>
+                        <div class="works-date-item">
+                            <h5 class="works-date">2018</h5>
+                            <h4 class="works-type">Illustration</h4>
+                        </div>
+                        <p>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia
+                            consequat
+                            duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</p>
+                    </div>
                 </div>
-                <div class="works">
-                    <img src="img/poster3.jpg" alt="img3">
-                    <h2>36 Days of Malayalam type</h2>
-                    <p>2018</p>
-                    <p>Typography</p>
-                    <p>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat
-                        duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</p>
+                <div class="line-h"></div>
+                <div class="works-list-item">
+                    <div class="works-img">
+                        <img src="img/poster3.jpg" alt="img3">
+                    </div>
+                    <div class="works">
+                        <h2>36 Days of Malayalam type</h2>
+                        <div class="works-date-item">
+                            <h5 class="works-date">2018</h5>
+                            <h4 class="works-type">Typography</h4>
+                        </div>
+                        <p>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia
+                            consequat
+                            duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</p>
+                    </div>
                 </div>
+                <div class="line-h"></div>
             </div>
-        </div>
     </section>
     <footer class="footer">
         <div class="container">
             <div class="footer-social">
-                <a href=""><img src="img/facebook.svg" alt="fb"></a>
-                <a href=""><img src="img/instagram.svg" alt="insta"></a>
-                <a href=""><img src="img/Twitter.svg" alt="Group"></a>
-                <a href=""><img src="img/Linkedin.svg" alt="Linkedin"></a>
+                <a href="#"><img src="img/facebook.svg" alt="fb"></a>
+                <a href="#"><img src="img/instagram.svg" alt="insta"></a>
+                <a href="#"><img src="img/Twitter.svg" alt="Group"></a>
+                <a href="#"><img src="img/Linkedin.svg" alt="Linkedin"></a>
             </div>
             <div class="footer-copyright">
                 <p>Copyright ©2020 All rights reserved</p>
```

Comment:
Replace © with `&copy;`, in some cases browser can't render such symbols. So you need to use HTML unicode for it